cgotorch/cgotorch.cc

```diff
@@ -37,6 +37,19 @@ char *RandN(int rows, int cols, int require_grad, Tensor *result) {
   }
 }
 
+char *RandNByShape(int64_t *shape, int64_t shape_len, int require_grad,
```

Comment:
`RandN` is supported by shape in #74 . We could remove `RandNByShape` after #74 is merged